src/main/java/crazypants/enderzoo/entity/EntityEnderminy.java

```diff
@@ -123,7 +123,7 @@ public boolean getCanSpawnHere() {
    */
   private boolean shouldAttackPlayer(EntityPlayer player) {
 
-    ItemStack itemstack = player.inventory.armorInventory[3];
+    ItemStack itemstack = player.inventory.armorInventory.get(3);
```

Comment:
Yep so its //    3: Helmet, 2: Chestpiece, 1: Legs, 0: Boots

Also i checked EntityEnderman.class and it says the same thing

`ItemStack itemstack = (ItemStack)player.inventory.armorInventory.get(3);

        if (itemstack.getItem() == Item.getItemFromBlock(Blocks.PUMPKIN))
        {
            return false;
        }`